src/connectedmachine/azext_connectedmachine/generated/_help.py

```diff
@@ -0,0 +1,127 @@
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['connectedmachine machine'] = """
+    type: group
+    short-summary: connectedmachine machine
+"""
+
+helps['connectedmachine machine list'] = """
+    type: command
+    short-summary: "Lists all the hybrid machines in the specified subscription. Use the nextLink property in the \
+response to get the next page of hybrid machines."
+    examples:
+      - name: List Machines by resource group
+        text: |-
+               az connectedmachine machine list --resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine show'] = """
+    type: command
+    short-summary: "Retrieves information about the model view or the instance view of a hybrid machine."
+    examples:
+      - name: Get Machine
+        text: |-
+               az connectedmachine machine show --name "myMachine" --resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine delete'] = """
+    type: command
+    short-summary: "The operation to remove a hybrid machine identity in Azure."
+    examples:
+      - name: Delete a Machine
+        text: |-
+               az connectedmachine machine delete --name "myMachine" --resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine-extension'] = """
+    type: group
+    short-summary: connectedmachine machine-extension
+"""
+
+helps['connectedmachine machine-extension list'] = """
+    type: command
+    short-summary: "The operation to get all extensions of a non-Azure machine"
+    examples:
+      - name: GET all Machine Extensions
+        text: |-
+               az connectedmachine machine-extension list --machine-name "myMachine" --resource-group \
+"myResourceGroup"
+"""
+
+helps['connectedmachine machine-extension show'] = """
+    type: command
+    short-summary: "The operation to get the extension."
+    examples:
+      - name: GET Machine Extension
+        text: |-
+               az connectedmachine machine-extension show --machine-name "myMachine" --name "CustomScriptExtension" \
+--resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine-extension create'] = """
+    type: command
+    short-summary: "The operation to create or update the extension."
+    examples:
+      - name: Create or Update a Machine Extension
+        text: |-
+               az connectedmachine machine-extension create --machine-name "myMachine" --name "CustomScriptExtension" \
+--location "eastus2euap" --type "CustomScriptExtension" --publisher "Microsoft.Compute" --settings \
+"{\\"commandToExecute\\":\\"powershell.exe -c \\\\\\"Get-Process | Where-Object { $_.CPU -gt 10000 }\\\\\\"\\"}" \
+--type-handler-version "1.10" --resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine-extension update'] = """
+    type: command
+    short-summary: "The operation to create or update the extension."
+    examples:
+      - name: Create or Update a Machine Extension
```

Comment:
Remove `create or` here.

Reply:
addressed in: https://github.com/Azure/azure-rest-api-specs/pull/10795

Reply:
Are you going to regenerate the extension based on the new swagger? I suggest you can manually fix it in CLI first.

Reply:
I manually fixed this in the last commit